build/runtime.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+require 'phar://flow-php.phar/vendor/autoload.php';
+
+use Flow\ETL\DataFrame;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\SingleCommandApplication;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+(new SingleCommandApplication())
+    ->setName('Flow-PHP - Extract Transform Load - Data processing framework')
+    ->setVersion(FlowVersion::getVersion())
+    ->addArgument('input-file', InputArgument::REQUIRED, '')
+    ->setCode(function (InputInterface $input, OutputInterface $output): int {
+        $style = new SymfonyStyle($input, $output);
+
+        $filename = (string) $input->getArgument('input-file');
+        if (!\file_exists($filename)) {
+            $style->error("Input file ($filename) doesn't exists!");
+
+            return Command::FAILURE;
+        }
+
+        if (!\str_ends_with($filename, '.php')) {
+            $style->error("Input file must be a PHP one!");
+
+            return Command::FAILURE;
+        }
+
+        $pipeline = include $filename;
+        if (!$pipeline instanceof DataFrame) {
+            $style->error('Expecting Flow-PHP DataFrame, received: ' . (is_object($pipeline) ? get_class($pipeline) : gettype($pipeline)));
+
+            return Command::FAILURE;
+        }
```

Comment:
This logic could be extracted to standalone DataFrameLoader which can be properly covered with tests

Reply:
once this is extracted to standalone class (it can be passed by constructor with default implementation) you can also put a simple test for this command and pass DataFrameLoader as a mock

Reply:
Where do you see this new class? Cause it's not a real loader like we have already in ETL, so implementing the `Loader` interface seems wrong (as we don't have context nor rows).